### PR TITLE
Feat : Landing_Page

### PR DESCRIPTION
**Pull request by bipin7yadav**
added  Landing Page
added Home Page
added  Error Page

### CODE REVIEW

**Review by techieshubham30 — APPROVED** [review]
remove comments if not needed, you have not added any preview link although code looks good, good to merge.

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
+
 import './App.css';
+import { Header,Footer } from './components';
+import {Error} from "./pages/404/error"
+import { Routes,Route } from 'react-router-dom';
+import { LandingPage,Category } from './pages';
```
Comment (msdeshmukh009) [inline src/App.js:6]: We can do barrel export for pages which will be useful in managing the imports.

File: src/components/header/header.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.header{
+    width: 100%;
+    /* border: 1px solid black; */
```
Comment (msdeshmukh009) [inline src/components/header/header.css:3]: You can remove this comment if not required.

File: src/pages/404/error.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import "./error.css"
+function Error(){
```
Comment (msdeshmukh009) [inline src/pages/404/error.js:2]: We can name this component a little better. As `error.js` is somewhat confusing. We can also name it `NotFound.js`.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

**Pull request merged**